<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_opt">
      <attribute name="title">
       <string>Opções</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>161</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Saída</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Reachbility graph/set</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Markov chain</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Derivative of CTMC</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>Probabilities</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>Probabilities DTMC</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBox_7">
          <property name="text">
           <string>Dot graph language</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_an">
      <attribute name="title">
       <string>Análise Numérica</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Análise</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Steady-state</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Transient</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>Both</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Sensitivity</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Parâmetros de solução</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Approach</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Steady-state method</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_3"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Transient method</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox_4"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Maximum # iterations</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Minimum precision</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>M0 return rate</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="checkBox_12">
            <property name="text">
             <string>Compute cumulative probabilities</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>202</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="checkBox_13">
            <property name="text">
             <string>Steady-state detection</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>202</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_8">
            <property name="text">
             <string>2000</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_9">
            <property name="text">
             <string>0.000001</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_10">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Condições de parada</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_8">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Absorbing marking</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_9">
            <property name="text">
             <string>Transient vanishingh loops</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_10">
            <property name="text">
             <string>Transient initial marking</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_11">
            <property name="text">
             <string>Vanishing initial value</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Eliminação</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Vanishing markings</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_sim">
      <attribute name="title">
       <string>Simulação</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_18">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Método de simulação</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_8"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_4">
            <property name="text">
             <string># replications</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Number of replication / batches</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_5">
            <property name="text">
             <string>Error given</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2" rowspan="2">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QCheckBox" name="checkBox_14">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>201</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Output usual cumulative measures</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Required confidence for simulation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_17"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Length of each simulation iteration/batch</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Used for comparing the fluid places content</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="text">
             <string>0.000001</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Used for comparing the firing time conflict</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_5">
            <property name="text">
             <string>0.000001</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Seed of the random generator</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_6">
            <property name="text">
             <string>52836</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBox_29">
            <property name="text">
             <string>Print the report in the .out file in simulation.</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Warm-up time</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_7">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_analysis">
        <property name="text">
         <string>Análise numérica</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rb_simulation">
        <property name="text">
         <string>Simulação</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
